tools/s2i-dpdk/README.md

```diff
@@ -0,0 +1,36 @@
+
+# Basic S2I builder image for dpdk workloads
+
+This folder contains files for building a s2i dpdk base image.
+
+This folder also contains a simple dpdk workload test inside the `test/test-app` folder.
+The test application use `expect` to run `testpmd`.
+
+# How to use
+
+To use this s2i you only need to run the [build-config.yaml](base-image/build-config.yaml).
+
+This will create a new "dpdk-testing" namespace then it will configure a "ImageStream" for the image and start a build. 
+
+After the build is ready you can edit the dpdk demo folder under the feature-configs.
```

Comment:
Can we suggest to build another layer instead of editing a file?

Reply:
Done :)